datalad_hirni/commands/import_additional_data.py

```diff
@@ -0,0 +1,168 @@
+            no dataset is given, an attempt is made to identify the dataset
+            based on the current working directory and/or the `path` given""",
+            constraints=EnsureDataset() | EnsureNone()),
+        path=Parameter(
+            args=("path",),
+            metavar='PATH',
+            doc="""path(s) of the data to be imported.""",
+            nargs="*",
+            constraints=EnsureStr()),
+        acqid=Parameter(
+            args=("acqid",),
+            metavar="ACQUISITION ID",
+            doc="""acquisition identifier for the imported data files""",
+            constraints=EnsureStr() | EnsureNone()),
+        target_dir=Parameter(
+            args=("-t", "--target-dir"),
+            metavar="TARGET",
+            doc="""subdirectory of the acquisition to store the data in""",
+            constraints=EnsureStr() | EnsureNone()),
+
+    )
+
+    @staticmethod
+    @datasetmethod(name='hirni_import_data')
+    @eval_results
+    def __call__(acqid, path, dataset=None, target_dir=None):
+
+        ds = require_dataset(dataset, check_installed=True,
+                             purpose="import additional data")
+        acq_dir = op.join(ds.path, acqid)
+        if not op.exists(acq_dir):
+            raise ValueError("Acquisition %s does not yet exist" % acqid)
+
+        path = assure_list(path)
+
+        # TODO: spec file specifiable or fixed path?
+        #       if we want the former, what we actually need is an association
+        #       of acquisition and its spec path
+        #       => prob. not an option but a config
+        spec_path = op.join(ds.path, acqid, "studyspec.json")
+        spec = [r for r in json_py.load_stream(spec_path)] \
+            if op.exists(spec_path) else list()
+
+        ds_meta = ds.metadata(reporton='datasets', return_type='generator',
+                              result_renderer='disabled')
+
+        for p in path:
+
+            src = RI(p)
+            if isinstance(src, PathRI):
+                src = RI("file://" + PathRI(op.abspath(p)).posixpath)  # TODO: Is this the right way for file-scheme on windows?
+
+            dst = op.join(acq_dir, target_dir if target_dir else "",
+                          op.basename(src.path))  # TODO: Again windows: If an actual URL (POSIX-path), does it still work with os.path.basename?
+            lgr.debug("Adding %s to %s", src, dst)
+
+            result = {'action': 'import additional data',
+                      'type': 'file',
+                      'path': dst,
+                      'logger': lgr}
+
+            if op.lexists(dst):
+                yield result.update({'status': 'impossible',
+                                     'message': 'file already exists'})
+                return
+
+            annex_result = ds.repo.repo.add_url_to_file(file_=dst, url=src,
+                                                        unlink_existing=False)
```

Comment:
I don't know about that. We will add URL to local files that are almost guaranteed to be broken few secs later. Would need to plan with it to see how much trouble this causes.

Reply:
I'm not sure either. That's just the general case, but we certainly need to figure out when and how to do it differently.